sources/packages/web/src/views/aest/student/StudentApplicationChangeRequests.vue

```diff
@@ -0,0 +1,194 @@
+<template>
+  <full-page-container :full-width="true">
+    <template #header>
+      <header-navigator title="Student requests" subTitle="Change requests" />
```

Comment:
Minor question to check with business. Will it be a good idea to match the title with link description ? 

![image](https://github.com/user-attachments/assets/f133fe24-e0d1-4e10-97ad-32a613678ff1)

If business think it is not required, then we can ignore it.